### PR TITLE
:rocket: Migrate to kontinuous

### PR DESCRIPTION
**Pull request by Matmut7**
Notes :
- **domaines hors Fabrique** : à tester

### CODE REVIEW

File: .kube-workflow/templates/app.sealed-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  annotations:
+    sealedsecrets.bitnami.com/cluster-wide: 'true'
+  name: app
+spec:
+  encryptedData:
+    PRISMIC_ACCESS_TOKEN: AgB4z2bRS3T3rE7y6eZWNs07EYANLhdM9V+Pipo0aVJy4OappAMAyRdDHgqmk2VWastr9j5ABdfCnG8DDeFKKY/Z9WNQ/u0jVv2RVkv1ffHaMBE12Wjydqyc1C9kQRM+QG0HAhGLN1j6/094pdgLgL6CKPeJxmCiHyf3zIP/ZvMqE6WyYX0YMDGHWThDXZoB1wcmrqQ9G+kh+WupCZO1P5tJbLvlqGgymk6MjYEN9UiNWJ8h4Jm6+59OZZPYUsVl/lylIS2t9c5f8LMhEEKdBQs23QLpWQ0jjACKGFiJscrLHbHrmCZ2g+Xo9Pj7finYAIHBkIKlOdKeRphGmsSw1Txq4mfdCQuUcocjqMqxqu/sDUKLBBLuS0bCsV9NyAR29t9PwK8X3UAQLLErsS7NXN1vmA03oGg7RKwrSuAtERyP/EAImDGQhwmKmyv994VxSjVEkSifHBPxPjf5uoOVFkCb9QE1skSLOqQM7ON47hsMulWdDCi3NTxNB7yEHyk0/ZZDETFaE6SNo/MEQ00jUPNoyPu7km2E4G8dR6zrUbUKozNxt2uQyqQQ9uV/jFIHt7ql9cuXg7+EMc2H/HepysvsFdQLrZonDIritJYQ6orFM18nWbeYDvq39F0niKknfR5vJFXPS4FMCiov2UPijBZEELR+L5svYCM2k2/RdVUi4ubBlW51kAUj7HDA/hoLaodPLiSvVnopYq9nf8/rS5uuJFKk4+V6yZo0kyOr/PqKH2RCi85lBuIEIEzz0w==
```
[the remark this answers is not in the document]
Comment (Matmut7) [reply .kube-workflow/templates/app.sealed-secrets.yaml:9]: Tu as le token Prismic dans les secrets Github ? Notre manière standard de faire serait plutôt un sealed secret comme celui-là

File: package.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
     "react-dom": "18.1.0",
     "react-markdown": "^8.0.3",
     "sass": "^1.50.1",
+    "sharp": "^0.30.5",
```
[the remark this answers is not in the document]
Comment (Matmut7) [reply package.json]: Le paquet est retiré pour le moment

File: .github/workflows/preproduction.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-latest
+    needs: [register]
+    environment:
+      name: preproduction
+      url: https://staging.archifiltre.fr
+    steps:
+    - name: Use kube-workflow deployment
+      uses: SocialGouv/kube-workflow@master
+      with:
+        environment: preprod
```
[the remark this answers is not in the document]
Comment (Matmut7) [reply .github/workflows/preproduction.yml:35]: Ce n'est pas automatique
Une proposition est désormais dans la PR f0339b6

File: .github/workflows/ks-preproduction.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: Preproduction
+on:
+  push:
+    branches:
+      - master
+      - main
```
Comment (lsagetlethias) [inline .github/workflows/ks-preproduction.yml:6]:
```suggestion
    branches:
      - dev
      - main
```

File: yarn.lock [hunk 5]
```diff
@@ -2761,16 +2761,32 @@ color-name@1.1.3:
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.3.tgz#a7d0558bd89c42f795dd42328f740831ca53bc25"
```
Comment (lsagetlethias) [inline yarn.lock:2761]: You cannot commit lock file without package json when a dependency is added (in this case `sharp` I guess)

File: .github/workflows/ks-review.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+name: Review
+on:
+  push:
+    branches-ignore:
+      - master
+      - main
```
[the remark this answers is not in the document]
Comment (lsagetlethias) [reply .github/workflows/ks-review.yml:6]: mayby not relevant as dev is not considered as review branch in kontinuous

File: .github/workflows/ks-preproduction.yml (hunk 4 above)
Comment (lsagetlethias) [reply .github/workflows/ks-preproduction.yml:6]: mayby not relevant as dev is not considered as review branch in kontinuous

File: .kontinuous/values.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  runs:
+    build-app:
+      use: SocialGouv/kontinuous/plugins/fabrique/jobs/build@v1
+      with:
+        imageProject: archifiltre
+        imageRepository: archifiltre-site-v2
+        imagePackage: app
+        registrySecretRefName: harbor
+        # buildArgs:
+        #   NEXT_PUBLIC_APP_VERSION_COMMIT: "{{ $.Values.global.sha }}"
```
[the remark this answers is not in the document]
Comment (Matmut7) [reply .kontinuous/values.yml:20]: done

**Review by lsagetlethias — APPROVED** [review]

**Pull request merged**